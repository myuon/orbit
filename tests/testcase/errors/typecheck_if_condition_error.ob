fun main() do
  if 42 do
    return 1;
  end
end