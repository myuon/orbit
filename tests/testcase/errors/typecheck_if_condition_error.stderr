If condition must be boolean, got int